internal/core/window.rs

```diff
@@ -824,6 +829,15 @@ impl WindowInner {
     pub fn hide(&self) -> Result<(), PlatformError> {
         let result = self.window_adapter().set_visible(false);
         self.strong_component_ref.borrow_mut().take();
+        GLOBAL_CONTEXT.with(|ctx| {
+            if let Some(ctx) = ctx.get() {
+                let mut count = ctx.window_count.borrow_mut();
+                *count -= 1;
+                if *count <= 0 {
+                    let _ = crate::api::quit_event_loop();
```

Comment:
Nothing bad should happen, but it might be worth here to drop the RefMut before calling this function - to be on the safe side.